.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": true
```

Comment:
To who this may concern if they want additional things added to the settings for the workspace I am open to ideas

Reply:
"to whom" it may concern

Reply:
@benjaminpaige 
Let's not be sassy here